src/Catalyst.Core.Modules.Consensus.Tests/UnitTests/Deltas/DeltaBuilderTests.cs

```diff
@@ -113,220 +117,227 @@ public void BuildDeltaEmptyPoolContent()
+
+        //    var rawAndSaltedEntriesBySignature = publicEntries.Select(e =>
+        //        {
+        //            var contractEntriesProtoBuff = e;
+        //            return new
+        //            {
+        //                RawEntry = contractEntriesProtoBuff,
+        //                SaltedAndHashedEntry =
+        //                    _hashProvider.ComputeMultiHash(contractEntriesProtoBuff.ToByteArray().Concat(salt))
+        //            };
+        //        }).ToArray();
+
+        //    var shuffledEntriesBytes = rawAndSaltedEntriesBySignature
+        //       .OrderBy(v => v.SaltedAndHashedEntry.ToArray(), ByteUtil.ByteListComparer.Default)
+        //       .SelectMany(v => v.RawEntry.ToByteArray())
+        //       .ToArray();
+
+        //    var expectedBytesToHash = BuildExpectedBytesToHash(expectedSelectedTransactions, shuffledEntriesBytes);
+
+        //    var deltaBuilder = new DeltaBuilder(transactionRetriever, _randomFactory, _hashProvider, _peerSettings,
+        //        _cache, _dateTimeProvider, _mapperProvider, _logger);
+        //    var candidate = deltaBuilder.BuildCandidateDelta(_previousDeltaHash);
+
+        //    ValidateDeltaCandidate(candidate, expectedBytesToHash);
+
+        //    _cache.Received(1).AddLocalDelta(Arg.Is(candidate), Arg.Any<Delta>());
+        //}
+
+        private static PublicEntry[] BuildSelectedTransactions(List<PublicEntry> transactions)
         {
-            var selectedTransactions = transactions.Where(t => (t.IsPublicTransaction || t.IsContractCall || t.IsContractDeployment) && t.HasValidEntries()).ToArray();
-            return selectedTransactions;
+            return transactions.ToArray();
+            //var selectedTransactions = transactions.Where(t => (t.IsPublicTransaction || t.IsContractCall || t.IsContractDeployment) && t.HasValidEntries()).ToArray();
+            //return selectedTransactions;
         }
 
-        private byte[] BuildExpectedBytesToHash(TransactionBroadcast[] selectedTransactions, byte[] shuffledEntriesBytes)
-        {
-            var signaturesInOrder = selectedTransactions
-               .Select(p => p.Signature.ToByteArray())
-               .OrderBy(s => s, ByteUtil.ByteListComparer.Default)
-               .SelectMany(b => b)
-               .ToArray();
-
-            var expectedCoinBase = new CoinbaseEntry
-            {
-                Amount = selectedTransactions.Sum(t => t.SummedEntryFees()).ToUint256ByteString(),
-                ReceiverPublicKey = _producerId.PublicKey.ToByteString()
-            };
-
-            var expectedBytesToHash = shuffledEntriesBytes.Concat(signaturesInOrder)
-               .Concat(expectedCoinBase.ToByteArray()).ToArray();
-            return expectedBytesToHash;
-        }
+        //todo
+        //private byte[] BuildExpectedBytesToHash(TransactionBroadcast[] selectedTransactions, byte[] shuffledEntriesBytes)
+        //{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4562326)